controller/space.go

```diff
@@ -435,13 +435,24 @@ func deleteOpenShiftResource(
 			}
 		}
 	}
+
 	if len(errorsList) != 0 {
 		var errString string
 		for _, err = range errorsList {
 			errString += fmt.Sprintf("%s\n", err)
 		}
 		return errors.NewInternalErrorFromString(errString)
 	}
+
+	//delete pipelines from the space
+	spaceLabel := space.Data.Attributes.Name
+	log.Debug(ctx, map[string]interface{}{"label": spaceLabel}, "deleting pipelines in")
+	resp, err = cl.DeletePipelines(ctx, client.DeletePipelinesPath(), spaceLabel)
+
+	if err != nil {
+		log.Error(ctx, nil, fmt.Sprintf("error occurred while deleting pipelines from space : %s", spaceLabel))
+	}
```

Comment:
we need to return error here .. right ??

Reply:
Initially, I assumed it a failure of this call should not fail delete space [considering](https://github.com/fabric8-services/fabric8-wit/pull/1902#discussion_r169061490) .

I will fix this.